src/Versioned.php

```diff
@@ -2797,8 +2844,78 @@ public static function get_including_deleted($class, $filter = "", $sort = "")
         if (!empty($sort)) {
             $list = $list->orderBy($sort);
         }
-        $list = $list->setDataQueryParam("Versioned.mode", "latest_versions");
-        return $list;
+        return static::updateListToIncludeDeleted($list);
+    }
+
+    /**
+     * Update a DataList to query the latest version of each record stored in the (class)_Versions tables.
+     *
+     * In particular, this will query deleted records as well as active ones.
+     */
+    public static function updateListToIncludeDeleted(DataList $list): DataList
```

Comment:
```suggestion
    public static function updateDataListToAlsoIncludeDeleted(DataList $list): DataList
```
And update references to it

Reply:
Done - though `updateDataList` feels overly verbose, so kept as `updateList...`.

The "also" feels a bit redundant but I've kept it, I guess you intend it to constrast with the "only" from the other methods?